Translation/Extractor/Visitor/Translation/FormType/DefaultTranslationDomainExtractor.php

```diff
@@ -187,7 +188,7 @@ private function isDefaultsDeclaration(Node $node)
         return ($node instanceof Node\Expr\MethodCall
             && $node->var instanceof Node\Expr\Variable
             && $node->var->name === $this->optionsResolverName
-            && $node->name === self::SET_DEFAULTS_DECLARATION_METHOD_NAME
+            && (string) $node->name === self::SET_DEFAULTS_DECLARATION_METHOD_NAME
```

Comment:
No need to cast to string if we are using `===` to compare with a string.

Reply:
I need to in order to trigger the implemented method __toString() of the object.

See https://github.com/nikic/PHP-Parser/blob/master/UPGRADE-4.0.md#changes-to-the-node-structure for more information